ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -0,0 +1,198 @@
+          return output.avoidHealerTower!();
+      },
+      outputStrings: {
+        avoidTankTower: {
+          en: 'Avoid Tank Tower',
+        },
+        avoidDpsTower: {
+          en: 'Avoid DPS Towers',
+        },
+        avoidHealerTower: {
+          en: 'Avoid Healer Tower',
+        },
+      },
+    },
+    {
+      id: 'P4N Northerly Shift Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4A', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go North, Stand on sides',
+        },
+      },
+    },
+    {
+      id: 'P4N Easterly Shift Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4C', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go East, Stand on sides',
+        },
+      },
+    },
+    {
+      id: 'P4N Southerly Shift Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4B', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go South, Stand on sides',
+        },
+      },
+    },
+    {
+      id: 'P4N Westerly Shift Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4D', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go West, Stand on sides',
+        },
+      },
+    },
+    // Call outs can be adjusted to either merge all to one or give them directional knockback call out
+    {
+      id: 'P4N Northerly Shift Knockback',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6DAE', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Easterly Shift Knockback',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6DB0', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Southerly Shift Knockback',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6DAF', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Westerly Shift Knockback',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6DB1', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Belone Bursts',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69D9', source: 'Hesperos', capture: false }),
+      alarmText: (data, _matches, output) => {
```

Comment:
Same here, that alarmText seems like a lot.  Maybe infoText?

Reply:
Still unresolved.

Reply:
This one as well in 296760177934d1b86b7ea5a8749618ce8a354173